LDKNodeMonday/View Model/Home/OnboardingViewModel.swift

```diff
@@ -8,23 +8,23 @@
 import LDKNode
 import SwiftUI
 
-class OnboardingViewModel: ObservableObject {
-    @AppStorage("isOnboarding") var isOnboarding: Bool?
-    @Published var networkColor = Color.gray
-    @Published var onboardingViewError: MondayError?
-    @Published var seedPhrase: String = "" {
+@Observable
+class OnboardingViewModel {
+    var networkColor = Color.gray
+    var onboardingViewError: MondayError?
+    var seedPhrase: String = "" {
```

Comment:
change requested: I like @Observable more too, but since we AppStorage and @Observable dont play nice together I see why you did what you did. 

however I'd prefer to leave this view model using `ObservableObject` and `@AppStorage("isOnboarding") var isOnboarding: Bool?` for the moment just because we do the same thing in 2 other view models, so if we make a change to this one we should make a change to the other 2, and for the moment I'd prefer we just keep things as is to keep this PR focused (unless you just couldn't get things to work with ObservableObject at all? I know you mentioned something about ObservableObject related to this PR, so let me know if ObservableObject just didn't work for you anymore here?)

I do like using `@Observable` and I also like using `AppStorage` so we will have to come back to this in another PR to get those two working together.

Reply:
Managed to switch back to @ObservableObject, clearly the errors I encountered previously were user errors.
This minimises the changes for this PR.

I do wonder however for the future if we can find another pattern to rely on when knowing if the app should be in onboarding mode or not, other than calling Appstorage from so many different files. Some options:

- If a node is running successfully in LightningNodeService, we are not in onboarding mode
- If BackupInfo is found (for the relevant network) node should start and we are not in onboarding mode

Here's what I did in BDG Wallet:

```.swift
// Check if user already has a node seed
if backupManager.backupInfo != nil {
   // If they do, start node
   do {
      try ldkNodeManager.start(mnemonic: backupManager.backupInfo!.mnemonic, passphrase: nil)
   } catch let error {
      debugPrint(error)
   }
}
```